frontend/public/components/dashboard/dashboards-page/overview-dashboard/details-card.tsx

```diff
@@ -17,11 +20,67 @@ import {
   getK8sGitVersion,
   ClusterVersionKind,
   getClusterID,
+  getDesiredClusterVersion,
+  getLastCompletedUpdate,
+  getClusterUpdateStatus,
+  getClusterVersionChannel,
+  ClusterUpdateStatus,
+  getOCMLink,
 } from '../../../../module/k8s';
 import { FLAGS } from '../../../../const';
 import { flagPending, featureReducerName } from '../../../../reducers/features';
-import { FirehoseResource } from '../../../utils';
+import { FirehoseResource, ExternalLink, history } from '../../../utils';
 import { RootState } from '../../../../redux';
+import { clusterUpdateModal } from '../../../modals';
+
+const ClusterVersion: React.FC<ClusterVersionProps> = ({ cv }) => {
+  const desiredVersion = getDesiredClusterVersion(cv);
+  const lastVersion = getLastCompletedUpdate(cv);
+  const status = getClusterUpdateStatus(cv);
+
+  switch (status) {
+    case ClusterUpdateStatus.Updating:
+      return (
+        <>
+          <span className="co-select-to-copy">{desiredVersion}</span>
+          <div>
+            <Button
+              variant="link"
+              className="btn-link--no-btn-default-values"
+              onClick={() => history.push('/settings/cluster/')}
```

Comment:
Make this a link instead of a button since it's a link :)

Reply:
done